modules/ROOT/pages/dw-core-functions-find.adoc

```diff
@@ -3,81 +3,121 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-//* <<find1>>
-//* <<find2>>
-//* <<find3>>
-
 
 [[find1]]
 == find(Array<T>, Any): Array<Number>
 
-Returns the array of index where the element to be found where present
+Returns the indices where a specified `elementToFind` is present.
+
+
+=== Parameters
+
+[%header, cols="1,3"]
+|===
+| Name   | Description
+| `elements` | An array (`Array` type).
```

Comment:
We haven't specified the type of the array anywhere else. Is this the only place where it matters? Are there arrays that aren't of type Array?